docs/docs/_index.md

```diff
@@ -114,64 +117,90 @@ OK
    3) "loggedOut"
 ```
 
-#### Python example
+If you run `redis-cli` in raw output mode, you can format the JSON returned by `JSON.GET` to make JSON objects more readable:
```

Comment:
You can also control the format in regular non-raw mode, but the control characters are printed "properly" (pretty/for humans) in raw mode.

Reply:
I changed the text slightly to focus on the human-readable part.